src/components/boardComponents/index.jsx

```diff
@@ -35,7 +35,7 @@ export const itemMap = {
     component: Rect,
     defaultActions: ["lock", "remove"],
     availableActions: [
-      "stack",
+      "stackToTopLeft",
```

Comment:
I think you must keep the name "stack" to keep compatibility with already existing games.